virtool/indexes/tasks.py

```diff
@@ -89,7 +89,7 @@ async def _add_one_json(self, index: Document):
 
             await run_in_thread(
                 compress_json_with_gzip,
-                json.dumps(
+                orjson.dumps(
```

Comment:
Just use your custom `dumps`. What is `"targets"` gets a `datetime` in it at some point.